src/cache-tag.ts

```diff
@@ -0,0 +1,6 @@
+import { GetCacheTag } from "./";
+
+export const akamaiGetCacheTag: GetCacheTag = ({ res }) => {
+  const header = res.getHeader("Edge-Cache-Tag");
+  return header ? String(header) : undefined;
+};
```

Comment:
```typescript
export const akamaiGetCacheTag: GetCacheTag = ({ res }) => `${res.getHeader("Edge-Cache-Tag")}`;
```

Reply:
I typed the return of the function to override express' mistype on the get function.